addons/note/models/note_note.py

```diff
@@ -0,0 +1,112 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from openerp import api, fields, models
+from openerp.tools import html2plaintext
+
+class NoteNote(models.Model):
```

Comment:
Some remarks 
1/ put compute fields after the field declaration
2/ put them in the fields declaration order